Code/C++/binary_tree_BFS_traversal.cpp

```diff
@@ -0,0 +1,118 @@
+/*Binary tree is a data structure that has at most 2 child node i.e. left and right child
```

Comment:
also, update spacing here as 
/*
TEXT....
*/

not like 
/* TEXT */

for better indentation

Reply:
done sir, please review it